src/App.css

```diff
@@ -39,9 +39,17 @@
     align-items: center;
 }*/
 
+.screen {
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
 .square {
-    width: 32.32%;
-    height: 32.32%;
+    width: 32.18%;
+    height: 32.18%;
```

Comment:
At some point in a separate PR, your css logic for how to render squares needs to be completely redone.